processor/samplingprocessor/tailsamplingprocessor/processor.go

```diff
@@ -131,11 +131,45 @@ func getPolicyEvaluator(logger *zap.Logger, cfg *PolicyCfg) (sampling.PolicyEval
 	case RateLimiting:
 		rlfCfg := cfg.RateLimitingCfg
 		return sampling.NewRateLimiting(logger, rlfCfg.SpansPerSecond), nil
+	case Composite:
+		rlfCfg := cfg.CompositeCfg
+		return newCompositeFromCfg(logger, rlfCfg)
 	default:
 		return nil, fmt.Errorf("unknown sampling policy type %s", cfg.Type)
 	}
 }
 
+func newCompositeFromCfg(logger *zap.Logger, config CompositeCfg) (sampling.PolicyEvaluator, error) {
+	var SubPolicyEvalParams []sampling.SubPolicyEvalParams
+	rateAllocationsMap := getRateAllocationMap(config)
+	for i := range config.PolicyCfgs {
+		policyCfg := &config.PolicyCfgs[i]
+		policy, _ := getPolicyEvaluator(logger, policyCfg)
+
+		evalParams := sampling.SubPolicyEvalParams{
+			Evaluator:         policy,
+			MaxSpansPerSecond: int64(rateAllocationsMap[policyCfg.Name]),
+		}
+		SubPolicyEvalParams = append(SubPolicyEvalParams, evalParams)
+	}
+	return sampling.NewComposite(logger, config.MaxTotalSpansPerSecond, SubPolicyEvalParams, sampling.MonotonicClock{}), nil
+}
+
+func getRateAllocationMap(config CompositeCfg) map[string]float64 {
```

Comment:
Add a comment to explain the purpose of the func.

Reply:
I will be opening new PR for https://github.com/open-telemetry/opentelemetry-collector/issues/1306 in next few days. I will also include composite policy in that change.